force-app/main/default/classes/SOQL.cls

```diff
@@ -345,17 +440,34 @@ public with sharing class SOQL {
      */
     public static void setPrimaryContact() {
         //Use the getOpportunityDetailsForClosedWon() method to get the Opportunities
+        List<Opportunity> opportunities = SOQL.getOpportunityDetailsForClosedWon();
```

Comment:
Since the method exists inside this same class, you don't need the class name. 
```suggestion
        List<Opportunity> opportunities = getOpportunityDetailsForClosedWon();
```